pyquil/simulation/tests/test_reference_wavefunction.py

```diff
@@ -921,10 +921,10 @@ def include_measures(request):
     return request.param
 
 
-def test_vs_lisp_qvm(n_qubits, prog_length):
+def test_vs_lisp_qvm(n_qubits, prog_length, client):
```

Comment:
type annotations please (they may not have been there but we can slowly improve that)